Modules/Utils.cs

```diff
@@ -1954,8 +1954,8 @@ public static void ApplySuffix(PlayerControl player)
                     _ => name
                 };
             }
-            
-            if (!name.Contains('\r') && player.FriendCode.GetDevUser().HasTag() && (player.AmOwner || player.IsModClient()))
+
+            if (!name.Contains($"\r\r") && player.FriendCode.GetDevUser().HasTag())
```

Comment:
Same here

Reply:
this was intentional so it shows the tag even if you haven't the mod running